python/instrumentation/openinference-instrumentation-beeai/src/openinference/instrumentation/beeai/__init__.py

```diff
@@ -13,136 +13,122 @@
 # limitations under the License.
 
 import logging
-from importlib import import_module
 from importlib.metadata import PackageNotFoundError, version
 from typing import Any, Callable, Collection, TypeVar
 
-import wrapt
+from beeai_framework.context import (
+    RunContextFinishEvent,
+)
+from beeai_framework.emitter import Emitter, EmitterOptions, EventMeta
```

Comment:
With Otel instrumentation it's typical to lazily import so that the instrumentor doesn't cause an import error if you don't have the package. E.g. it can be bundled with other instrumentors and automatically be initialized when beeai gets added. Up to you ofc. but definitely a nice pattern.